public/content/developers/docs/data-and-analytics/index.md

```diff
@@ -1,55 +1,122 @@
 
-## Ethernow - Mempool Data Program {#ethernow}
-[Blocknative](https://www.blocknative.com/) provides open access to its Ethereum historical [mempool data archive](https://www.ethernow.xyz/mempool-data-archive). This enables researchers and community good projects to explore the pre-chain layer of Ethereum Mainnet. The data set is actively maintained and represents the most comprehensive historical record of mempool transaction events within the Ethereum ecosystem. Learn more at [Ethernow](https://www.ethernow.xyz/). 
+It’s common to jump between well-crafted data dashboards and blockchain explorers iteratively, to help you identify a trend or build out a cohesive data pipeline.
+
+Outside of etherscan, we now have this plethora of explorers to choose from:
+
+- More intuitive explorers like [parsec](https://parsec.fi/), [arkham](https://platform.arkhamintelligence.com/), [onceupon](https://www.onceupon.gg/home) showcasing more metadata and charts for transactions and addresses.
+- In depth explorers like [evm.storage](https://evm.storage/eth/18906826/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48#map) showcasing storage slots and memory stacks
+- Cross-chain explorers like [Routescan](https://routescan.io/) (superscan), [Dora](https://www.ondora.xyz/), and [Onceupon](https://www.onceupon.gg/home)
+- ZK proof explorers like [modulus](https://explorer.modulus.xyz/batch/157/inference/61974), [succinct](https://alpha.succinct.xyz/), and [axiom](https://explorer.axiom.xyz/v1/mainnet).
+- Bridge explorers like [socketscan](https://www.socketscan.io/) and [wormhole](https://wormhole.com/explorer/)
+- MEV explorers like [Eigenphi](https://eigenphi.io/mev/ethereum/txr) for transactions or [mevboost.pics](https://mevboost.pics/) for bundles and [beaconcha.in](https://beaconcha.in/) for blocks
+- Nansen launched a 2.0 of their token and wallet tracking product, with cool new features like “smart segments”
+
+The dashboard layer hasn’t changed much. It’s still the wild west here. If you spend a day on Twitter, you’ll see charts from dozens of different platforms covering similar data but all with slight differences or twists. Verification and attribution are becoming a bigger issue, especially now with the large growth in both teams and chains.
+
+Marketing specific address explorers are on the rise. Teams like spindl and bello will lead the way here. Cross-chain explorers (and pre-chain like MEV/intents) will see expansion in development.
+
+Across platforms, wallets are still poorly labelled and tracked, and it’s getting worse with intents/account abstraction now. We don’t mean static labels like “Coinbase” but instead more dynamic ones like “Experienced Contract Deployer”. Some teams are trying to tackle this such as [walletlabels](https://walletlabels.xyz/), [onceupon context](https://github.com/Once-Upon/context), and [syve.ai](https://www.syve.ai/). It will also improve naturally alongside web3 social, which is growing mainly on Farcaster.
```

Comment:
https://walletlabels.xyz
<img width="454" height="183" alt="image" src="https://github.com/user-attachments/assets/636d506a-cbf8-404f-8115-eca96041548a" />

<img width="605" height="509" alt="image" src="https://github.com/user-attachments/assets/60c5193e-d321-4b9c-b528-a0b7564ba48b" />

sad panda..

```suggestion
Across platforms, wallets are still poorly labelled and tracked, and it’s getting worse with intents/account abstraction now. We don’t mean static labels like “Coinbase” but instead more dynamic ones like “Experienced Contract Deployer”. Some teams are trying to tackle this, such as [syve.ai](https://www.syve.ai/). It will also improve naturally alongside web3 social, which is growing mainly on [Farcaster](https://farcaster.xyz/).
```